content/integrations/AWS-Integration/aws-cur.md

```diff
@@ -36,25 +36,28 @@ Already have an existing Cost and Usage Report with GZIP or ZIP compression set
 13. Select **Next**.
 ![final-steps](/images/aws-cur/final-steps.png)
 14. Review your configurations.
-15. Copy the **Report path prefix** found under Delivery Options (without /date-range/). You need this for part 2.
-![full-path](/images/aws-cur/full-path.png)
 16. Select **Review and Complete** to create the Cost and Usage Report (CUR).
 
 It can take up to a few hours for data to populate in the S3 bucket.
 
 ## 2. Update your AWS Integration in CloudWisdom
 
-If you have just created your IAM role, **wait 2-5 minutes** for AWS to finalize its creation before proceeding to these steps. This ensures the new role has the correct s3 access permissions when added to CloudWisdom.
+If you have just created your [IAM role][1], **wait 2-5 minutes** for AWS to finalize its creation before proceeding to these steps. This ensures the new role has the correct s3 access permissions when added to CloudWisdom.
 
 1. From the side navigation menu, select **Integrations**.
 2. Select the **Amazon Web Services** card.
-3. Select **Cost and Usage Report (recommended)** from the Detailed Billing Source dropdown.
-![detailed-billing-source](/images/aws-cur/detailed-billing-source.png)
-4. Scroll to the **Detailed Billing Settings** section and complete the following fields:
-   - **S3 Bucket Name**: the case sensitive name of bucket chosen in section 1, step 7.
-   - **Report Path Prefix**: Input the path prefix of an existing CUR or the new one created in section 1, step 8: `CostAndUsageReports/HourlyCSVWithResourceIDs` 
+3. Ensure both **Cost Explorer API** and **Detailed Billing** are enabled.
```

Comment:
Cost Explorer isn't part of CUR and the toggle name just changed to "Cost & Usage Reports"
```suggestion
3. Ensure **Cost & Usage Reports** is enabled.
```